prettyPrinters/providers.py

```diff
@@ -266,3 +266,93 @@ def update(self):
     def has_children(self):
         # type: () -> bool
         return True
+
+
+class StdCellSyntheticProvider:
+    """Pretty-printer for std::cell::Cell"""
+
+    def __init__(self, valobj, dict):
+        # type: (SBValue, dict) -> StdCellSyntheticProvider
+        logger = Logger.Logger()
+        self.valobj = valobj
+        self.value = valobj.GetChildMemberWithName("value").GetChildAtIndex(0)
+
+    def num_children(self):
+        # type: () -> int
+        return 1
+
+    def get_child_index(self, name):
+        # type: (str) -> int
+        if name == "value":
+            return 0
+        return -1
+
+    def get_child_at_index(self, index):
+        # type: (int) -> SBValue
+        if index == 0:
+            return self.value
+        return None
+
+    def update(self):
+        # type: () -> None
+        pass
+
+    def has_children(self):
+        # type: () -> bool
+        return True
+
+
+def StdRefCellSummaryProvider(valobj, dict):
+    # type: (SBValue, dict) -> str
+    assert valobj.GetNumChildren() == 2
+
+    borrow = valobj.GetChildMemberWithName("borrow").GetValueAsSigned()
+    return "borrow={}".format(borrow)
+
+
+class StdRefCellSyntheticProvider:
+    """Pretty-printer for std::cell::RefCell and std::cell::Ref"""
+
+    def __init__(self, valobj, dict, is_ref=False):
+        # type: (SBValue, dict) -> StdRefCellSyntheticProvider
+        logger = Logger.Logger()
+        self.valobj = valobj
+        self.borrow = valobj.GetChildMemberWithName("borrow").GetChildAtIndex(0).GetChildAtIndex(0)
+        if is_ref:
+            self.borrow = self.borrow.GetChildAtIndex(0)
+        self.value = valobj.GetChildMemberWithName("value").GetChildAtIndex(0)
+
+        process = valobj.GetProcess()
+        self.endianness = process.GetByteOrder()
+        self.pointer_size = process.GetAddressByteSize()
+        self.borrow_type = valobj.GetType().GetBasicType(eBasicTypeLong)
+
+    def num_children(self):
+        # type: () -> int
+        return 2
+
+    def get_child_index(self, name):
+        # type: (str) -> int
+        if name == "borrow":
+            return 0
+        if name == "value":
```

Comment:
I'd prefer to swap indices of `value` and `borrow` if we want to keep `borrow` child